scripts/check_changeset_completeness.ts

```diff
@@ -21,51 +27,80 @@ const getModifiedPackages = (changedFiles: string[]): Set<string> => {
   return modifiedPackageDirs;
 };
 
-const checkBackendDependenciesVersion = (releasePlan: ReleasePlan) => {
-  const backendDependencies: string[] = [
-    '@aws-amplify/backend-auth',
-    '@aws-amplify/backend-data',
-    '@aws-amplify/backend-function',
-    '@aws-amplify/backend-storage',
-  ];
-  const backendName: string = '@aws-amplify/backend';
-  const versionBumpOfWrongKind: string[] = [];
-  let backendMaxVersionType: string = 'none';
-
-  for (const changeset of releasePlan.changesets) {
-    for (const release of changeset.releases) {
-      if (release.name === backendName) {
-        if (
-          release.type === 'major' ||
-          backendMaxVersionType === 'none' ||
-          (backendMaxVersionType === 'patch' && release.type === 'minor')
-        ) {
-          backendMaxVersionType = release.type;
-        }
-      }
-    }
+const versionTypeConverter = (version: VersionType): VersionTypeEnum => {
```

Comment:
```suggestion
const convertVersionType = (version: VersionType): VersionTypeEnum => {
```

function name should be verb-ish.